no_short_enums.patch

From: Christian Prochaska <christian.prochaska@genode-labs.com>

Ensure -fno-short-enums as default.
---
 gcc/config/arm/arm.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/arm/arm.h b/gcc/config/arm/arm.h
index 2476cb3..da8196f 100644
--- a/gcc/config/arm/arm.h
+++ b/gcc/config/arm/arm.h
@@ -361,7 +361,7 @@ extern int arm_fpu_attr;
 /* AAPCS based ABIs use short enums by default.  */
 #ifndef ARM_DEFAULT_SHORT_ENUMS
 #define ARM_DEFAULT_SHORT_ENUMS \
-  (TARGET_AAPCS_BASED && arm_abi != ARM_ABI_AAPCS_LINUX)
+  (TARGET_AAPCS_BASED && arm_abi != ARM_ABI_AAPCS_LINUX && 0)
 #endif
 
 /* Map each of the micro-architecture variants to their corresponding
